test/e2e/app-dir/metadata/metadata.test.ts

```diff
@@ -671,7 +671,7 @@ createNextDescribe(
       it('should support root level of favicon.ico', async () => {
         let $ = await next.render$('/')
         const favIcon = $('link[rel="icon"]')
-        expect(favIcon.attr('href')).toBe('/favicon.ico')
+        expect(favIcon.attr('href')).toMatch('/favicon.ico')
```

Comment:
what will the full url of `/favicon.ico`? we expect `/favicon.ico` is the fixed one for backcompatible

Reply:
Turbopack currently puts a querystring after favicon.ico with its hashes. So if file's the same url would be probably remain as-is.